src/main/groovy/grails/plugin/sentry/SentryGrailsPlugin.groovy

```diff
@@ -40,15 +43,18 @@ class SentryGrailsPlugin extends Plugin {
     def documentation = 'http://github.com/agorapulse/grails-sentry/blob/master/README.md'
 
     def license = 'APACHE'
-    def developers = [[name: 'Benoit Hediard', email: 'ben@benorama.com']]
+    def developers = [[name: 'Benoit Hediard', email: 'ben@benorama.com'], [name: 'Alexey Zhokhov', email: 'alexey@zhokhov.com']]
     def issueManagement = [system: 'GitHub', url: 'http://github.com/agorapulse/grails-sentry/issues']
     def scm = [url: 'http://github.com/agorapulse/grails-sentry']
 
+    private SentryConfig savedConfig
```

Comment:
Actually not need to save it, can create `SentryConfig` every time on getting it.